prisma/seed.ts

```
@@ -35,6 +35,53 @@ async function main() {
     ),
   );
 
+  // Create at least 10 mocked events
+  const events = await Promise.all(
```

ESLint

[error] 'events' is assigned a value but never used.

(no-unused-vars)